conformance/reports/v1.0.0/kong-kubernetes-ingress-controller/README.md

```diff
@@ -5,5 +5,6 @@
 |API channel|Implementation version|Mode|Report|
 |-----------|----------------------|----|------|
 |x|[v3.0.2](https://github.com/Kong/kubernetes-ingress-controller/releases/tag/v3.0.2)|x|[link](./v3.0.2-report.yaml)|
+|x|[v3.1.1](https://github.com/Kong/kubernetes-ingress-controller/releases/tag/v3.1.1)|x|[link](./v3.1.1-report.yaml)|
 
 ## Reproduce
```

Comment:
Are we only going to start requiring this after the v1.1 release or should we just require this for all future conformance reports submitted, including this one?

Reply:
I don't see a big downside from just requiring this from here on out :thinking:

Reply:
Yes, I think it makes sense. I'll add the steps to reproduce 👍

Reply:
I've added the "reproduce" section.